modules/monitoring/files/usr/lib/nagios/plugins/check_reboots_required

```diff
@@ -1,10 +1,34 @@
 #!/usr/bin/env ruby
 
+def create_machines_hash
+  machines_list= `/usr/local/bin/govuk_node_list --list`.split("\n")
```

Comment:
I'd put a space before the `=` (so `machines_list = `/...`). Also, perhaps try `.lines` rather than `.split("\n")`.

You could maybe go even further, and do something the following and skip the `machines_list` variable:
```ruby
`/usr/local/bin/govuk_node_list ...`.each_line do |machine|
  ...
```

Reply:
great, I will do the last option.